Functions.Rmd

```diff
@@ -11,7 +11,7 @@ If you're reading this book, I expect that you've already created many R functio
 
 ### Quiz {-}
 
-Answer the following questions to see if you can safely skip this chapter. You can find the answers at the end of the chapter in [answers](#function-answers).
+Answer the following questions to see if you can safely skip this chapter. You can find the [answers](#function-answers) at the end of the chapter.
```

Comment:
We need to rewrite these to be (e.g.) in `Section \@ref(function-answer)` so they look the same in online and printed books.